### DISCUSSION TITLE
a community maintained repo of filament

### DISCUSSION THREAD

**Discussion by kinghat** [Donkie/Spoolman#371] [Ideas]
looking through issues and discussion there a bunch of half instances of this suggestion. what about the ability for spoolman to accept a public or private git repo that that would be a db of vendors and their filament. spoolman could have its own main db, much like themoviedb.org or how orca has a community supported db for printers and filaments. 

this wouldnt be a replacement for manually added vendors and filaments. users could use both. if implemented, there could be a feature to merged existing manually added vendors/filaments that match ones in the public repo or their own repos. the vendor and filament configs would probably be in json or a variant i would assume. im sure you've thought about something like this before.

**Comment by 2wenty2wo** [comment]
I was legit just looking for something like this. Would be helpful.

**Comment by Donkie** [comment]
Spoolman has such a repo nowadays, so I'm closing this.

https://github.com/Donkie/SpoolmanDB
